docs/client-libraries-tableviews.md

```diff
@@ -4,36 +4,105 @@ title: Work with TableView
 sidebar_label: "Work with TableView"
 ---
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+````
+
 After setting up your clients, you can explore more to start working with [TableView](concepts-clients.md#tableview).
 
 ## Configure TableView
 
-The following is an example of how to configure a TableView.
 
-```java
-TableView<String> tv = client.newTableViewBuilder(Schema.STRING)
-  .topic("my-tableview")
-  .create()
-```
+````mdx-code-block
+<Tabs groupId="lang-choice"
+defaultValue="Java"
+values={[{"label":"Java","value":"Java"},{"label":"C++","value":"C++"}]}>
+<TabItem value="Java">
+
+  The following is an example of how to configure a TableView.
+  ```java
+    TableView<String> tv = client.newTableViewBuilder(Schema.STRING)
+    .topic("my-tableview")
+    .create()
+  ```
+
+  You can use the available parameters in the `loadConf` configuration or related [API](/api/client/2.10.0-SNAPSHOT/org/apache/pulsar/client/api/TableViewBuilder.html) to customize your TableView.
+
+  | Name | Type| Required? |  <div>Description</div> | Default
+  |---|---|---|---|---
+  | `topic` | string | yes | The topic name of the TableView. | N/A
+  | `autoUpdatePartitionInterval` | int | no | The interval to check for newly added partitions. | 60 (seconds)
+  | `subscriptionName` | string | no | The subscription name of the TableView. | null
+
+</TabItem>
+
+<TabItem value="C++">
+
+
+  This feature is supported in C++ client 3.2.0 or later versions.
```

Comment:
```suggestion

:::note

This feature is supported in Pulsar 2.8.x and C++ client 3.2.0 or later versions.

:::

```

This feature is not server-independent, do we need to specify the Pulsar version?

Reply:
No, This feature just implements on the client side.